### PR TITLE
add client request ip

### PR DESCRIPTION
**Pull request by qAison**
1、access log add client request ip.
2、orm log delete repetition time.
3、const config dir and file.

### CODE REVIEW

File: utils/http.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"net/http"
+	"strings"
+)
+
+// 获取 客户端 IP
+func GetRequestIP(r *http.Request) string {
+	if ips := r.Header.Get("X-Forwarded-For"); ips != "" {
```
Comment (sergeylanzman) [inline utils/http.go]: 👍

File: router.go [hunk 2]
```diff
@@ -835,18 +835,19 @@ Admin:
 		statusColor := logs.ColorByStatus(iswin, statusCode)
 		methodColor := logs.ColorByMethod(iswin, r.Method)
 		resetColor := logs.ColorByMethod(iswin, "")
+		rip := utils.GetRequestIP(r)
```
Comment (astaxie) [inline router.go]: you can get the IP from `ctx.Input.IP()`. https://godoc.org/github.com/astaxie/beego/context#BeegoInput.IP

**Comment by astaxie** [comment]
@qAison leave the comments please update it

File: utils/http.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"net/http"
+	"strings"
+)
+
+// 获取 客户端 IP
```
Comment (xiaopeng163) [inline utils/http.go]: comments should be English I think.

Comment (astaxie) [inline utils/http.go]: the problem is we don't need this. As it's already implement in context.Input

**Comment by astaxie** [comment]
@qAison I have updated the code in develop. As you didn't update this PR as I suggested. I made these change.

1、access log add client request ip.
2、orm log delete repetition time.

 I will close this PR

**Pull request closed without merging**